web-app/modules/get-article-list.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import squidexApi from 'utils/squidex-api'
+
+class GetArticleList extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      articles: []
+    }
+  }
+
+  componentDidMount() {
+    squidexApi
```

Comment:
are we planning on using redux?  if so then this would be replaced with a `dispatch(getArticles()))` — I'll approve this in the short term but I'd rather see us move to using redux sooner than later

Reply:
Will be integrated in during ticket `DEV-187` after the CMS structure is finalized